pom.xml

```diff
@@ -2272,7 +2272,7 @@
     <properties>
 
         <!--Carbon Identity Framework Version-->
-        <carbon.identity.framework.version>7.3.41</carbon.identity.framework.version>
+        <carbon.identity.framework.version>7.3.41-SNAPSHOT</carbon.identity.framework.version>
```

Comment:
```suggestion
        <carbon.identity.framework.version>7.3.41</carbon.identity.framework.version>
```

Reply:
This will be updated when the relevant component is released.